cpuinfo.h

```diff
@@ -0,0 +1,6 @@
+#ifndef __CPUINFO_H__
```

Comment:
Why do we need two separate files with the same name and content?
There's another cpuinfo.h in WindowsStubs/WindowsStubs .